da/da_test.go

```diff
@@ -97,26 +92,21 @@ func TestSubmitRetrieve(t *testing.T) {
 	}
 }
 
-func startMockGRPCServ() *grpc.Server {
-	srv := proxy.NewServer(goDATest.NewDummyDA(), grpc.Creds(insecure.NewCredentials()))
-	lis, err := net.Listen("tcp", "127.0.0.1"+":"+strconv.Itoa(7980))
+func startMockDAServ() *proxy.Server {
+	srv := proxy.NewServer("localhost", "7980", goDATest.NewDummyDA())
+	err := srv.Start(context.TODO())
 	if err != nil {
-		fmt.Println(err)
-		return nil
+		panic(err)
 	}
-	go func() {
-		_ = srv.Serve(lis)
-	}()
 	return srv
 }
 
-func startMockGRPCClient() (*DAClient, error) {
-	client := proxy.NewClient()
-	err := client.Start("127.0.0.1:7980", grpc.WithTransportCredentials(insecure.NewCredentials()))
+func startMockDAClient() (*DAClient, error) {
+	client, err := proxy.NewClient(context.TODO(), "http://localhost:7980", "")
```

Comment:
same comment about passing in ctx from test and explaining the port.

Reply:
Fixed.